src/mutable_data.rs

```diff
@@ -244,7 +253,7 @@ macro_rules! impl_mutable_data {
                 }
             }
 
-            /// Insert or update permissions for the provided user.
+            /// Inserts or updates permissions for the provided user.
```

Comment:
We should mention what the `version` parameter means and that it must be equal to `current version + 1`.